### PR TITLE
Changing the IOperation unit test dumper to visit certain operation explicitly.

### PR DESCRIPTION
**Pull request by jinujoseph**
Addressing Followup feedback from https://github.com/dotnet/roslyn/pull/20716/files#diff-beae616c53aea23b7cc563565ff21f0eR278

### CODE REVIEW

**Comment by jinujoseph** [comment]
test windows_debug_unit32_prtest

**Comment by jinujoseph** [comment]
cc @dotnet/analyzer-ioperation 
@cston addressed your feedback, Pls review.

File: .cobyrc [hunk 1]
```diff
@@ -0,0 +1 @@
+solution-file-path: Roslyn.sln
```
Comment (cston) [inline .cobyrc]: Is this file addition intentional?

File: src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;
```
Comment (cston) [inline src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs]: Missing copyright header.

**Comment by cston** [comment]
LGTM

File: .cobyrc (hunk 1 above)
Comment (jinujoseph) [inline .cobyrc]: Good catch, it was not intentional , i have fixed it

File: src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs (hunk 2 above)
Comment (jinujoseph) [inline src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConversionExpression.cs]: fixed

**Pull request merged**